src/js/editable-element/controller.js

```diff
@@ -303,6 +304,9 @@ angular.module('xeditable').factory('editableController',
     //hide
     self.hide = function() {
 
+      // destroy the scope to prevent memory leak
+      self.editorEl.scope().$destroy();
```

Comment:
Definitely...  This code is not valid.  Thank you for the review.
I'm going to change it to reference a variable outside the function.